pan-domain-auth-example/app/di.scala

```diff
@@ -19,15 +19,24 @@ class AppComponents(context: Context) extends BuiltInComponentsFromContext(conte
   with AhcWSComponents
   with HttpFiltersComponents {
 
-  val awsCredentialsProvider = new StaticCredentialsProvider(new BasicAWSCredentials(
-    configuration.get[String]("aws.keyId"),
-    configuration.get[String]("aws.secret")
-  ))
+  val awsRegion = "eu-west-1"
+
+  // Change this to point to the S3 bucket containing the settings file
+  val bucketName = "pan-domain-auth-settings"
+
+  // Prefer profile credentials over static credentials
+  // If you work at the Guardian you can get these credentials from Janus
+  val awsCredentialsProvider = new AWSCredentialsProviderChain(
+    new ProfileCredentialsProvider("workflow"),
```

Comment:
If your goal is to decouple this from Guardian tooling does this still make sense?

Reply:
I've updated the example to have a more general note of advice:

```scala
// Customise as appropriate depending on how you manage your AWS credentials
```